eb_sqs/worker/service.py

```diff
@@ -4,8 +4,13 @@
 import signal
 from datetime import timedelta
 from time import sleep
+from typing import Any
 
 import boto3
+from boto.sqs.message import Message
```

Comment:
are these the right from packages? My PyCharm was complaining and I couldn't find an example of this anywhere.

Reply:
yeah i'm getting the same errors too, but when i look at the documentation, it looks correct - https://boto3.amazonaws.com/v1/documentation/api/latest/guide/sqs.html#sqs

Reply:
actually when i looked again boto = boto 2.0 and we're using boto 3.0 so the reference is incorrect. however, i can't find Queue nor Message in the boto3 (and botocore) packages. So I've left it out for now.